src/lib/mongoBackend/TriggeredSubscription.cpp

```diff
@@ -97,9 +99,14 @@ TriggeredSubscription::~TriggeredSubscription()
 * TriggeredSubscription class is shared for NGSI9 and NGSI10 subscriptions, so it is better
 * to keep expressions (an artifact for NGSI10) out of the constructor, in its independent fill
 * method
-*
 */
-void TriggeredSubscription::fillExpression (const std::string& q, const std::string& georel, const std::string& geometry, const std::string& coords)
+void TriggeredSubscription::fillExpression
+(
+  const std::string&  q,
+  const std::string&  georel,
+  const std::string&  geometry,
+  const std::string&  coords
+)
 {
```

Comment:
Related with my previous comment, this fillExpression() method should build the object pointed by stringFilterP (and the release() method should release the memory allocated here).